plugins/doc_fragments/pulp.py

```diff
@@ -38,6 +36,12 @@ class ModuleDocFragment(object):
       - If no value is specified, the value of the environment variable C(SQUEEZER_VALIDATE_CERTS) will be used as a fallback.
     type: bool
     default: true
+  refresh_api_cache:
+    description:
+      - Whether the cached api documentation should be invalidated.
+      - It is recomended to use this once with the M(status) module at the beginning of the playbook.
```

Comment:
```suggestion
      - It is recomended to use this once with the M(pulp.squeezer.status) module at the beginning of the playbook.
```

(or whatever the correct FQCN is)